polaris-service/src/test/java/org/apache/polaris/service/catalog/PolarisRestCatalogViewAzureIntegrationTest.java

```
@@ -0,0 +1,120 @@
+ * under the License.
+ */
+package org.apache.polaris.service.catalog;
+
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import org.apache.iceberg.rest.RESTCatalog;
+import org.apache.iceberg.view.ViewCatalogTests;
+import org.apache.polaris.core.admin.model.AzureStorageConfigInfo;
+import org.apache.polaris.core.admin.model.StorageConfigInfo;
+import org.apache.polaris.service.PolarisApplication;
+import org.apache.polaris.service.config.PolarisApplicationConfig;
+import org.apache.polaris.service.test.PolarisConnectionExtension;
+import org.apache.polaris.service.test.PolarisConnectionExtension.PolarisToken;
+import org.apache.polaris.service.test.PolarisRealm;
+import org.apache.polaris.service.test.SnowmanCredentialsExtension;
+import org.apache.polaris.service.test.SnowmanCredentialsExtension.SnowmanCredentials;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Import the full core Iceberg catalog tests by hitting the REST service via the RESTCatalog
+ * client.
+ */
+@ExtendWith({
+  DropwizardExtensionsSupport.class,
+  PolarisConnectionExtension.class,
+  SnowmanCredentialsExtension.class
+})
+public class PolarisRestCatalogViewAzureIntegrationTest extends ViewCatalogTests<RESTCatalog> {
+  public static final String TEST_TENANT_ID =
+      Optional.ofNullable(System.getenv("INTEGRATION_TEST_TENANT_ID"))
+          .orElse("xyz.onmicrosoft.com");
+  public static final String AZURE_BASE_PATH =
+      Optional.ofNullable(System.getenv("INTEGRATION_TEST_AZURE_PATH"))
+          .orElse("file:///tmp/buckets/my-bucket");
```

Since these are intended to be cloud-specific, I wonder if we should just fail or not run the test rather than falling back to local?

---

Updated the PR to be 1 file that takes environment variables to specify the cloud provider. I think the new way is cleaner than splitting the files and we still keep just the 1 test that runs locally.